views/pages/fixture_editor.js

```diff
@@ -991,37 +965,68 @@ function numberInput(modelKey, options = {}) {
 }
 
 /**
- * @param {!object} property The JSON schema property info.
- * @returns {?number} The lowest possible value the property can be set to, or null if there's no minimum.
+ * @param {!string} modelKey The Vue reference used in v-model.
+ * @param {!InputOptions} options Additional optional parameters.
+ * @returns {!string} The HTML attribute(s) declaring a minimum and/or maximum value, or an empty string.
  */
-function getMininium(property) {
-  if (property) {
-    if (`minimum` in property) {
-      return property.minimum;
-    }
-    if (`exclusiveMinimum` in property) {
-      return property.exclusiveMinimum;
+function getMinMaxAttributes(modelKey, options) {
+  let html = ``;
+
+  for (const minMax of [`in`, `ax`]) {
+    let minMaxValue = null;
+
+    if (options.property) {
```

Comment:
This check could better be done once before the loop: `if (!options.property) return '';`